assets/js/src/core/modules/element/dynamic-types/defintinitions/objects/data-related/types/abstract/dynamic-type-object-data-abstract-numeric.tsx

```diff
@@ -53,17 +64,27 @@ export abstract class DynamicTypeObjectDataAbstractNumeric extends DynamicTypeOb
     )
   }
 
-  getObjectDataComponentProps (props: AbstractNumericObjectDataDefinition): IInputNumberProps {
+  getVersionObjectDataComponentProps (props: AbstractNumericObjectDataDefinition): IInputNumberProps {
     return {
       inherited: props.inherited,
       disabled: props.noteditable === true,
       max: fixUnsigned(props.unsigned === true, props.maxValue, true) ?? undefined,
       min: fixUnsigned(props.unsigned === true, props.minValue, false) ?? undefined,
+      defaultValue: props.value,
       precision: props.integer === true ? 0 : (props.decimalPrecision ?? undefined),
-      step: props.increment ?? undefined
+      step: props.increment ?? undefined,
+      className: props.className
     }
   }
 
+  getVersionObjectDataComponent (props: AbstractNumericObjectDataDefinition): React.ReactElement<AbstractObjectDataDefinition> {
```

Comment:
Why do we need to override this method here? The abstract-input extends the regular input. Shouldn't be then our default implmentation from the `DynamicTypeObjectDataAbstract` be enough? In my opinion we do not need any changes in this file?